lms/static/sass/lms-footer-edx-rtl.scss.mako

```diff
@@ -1,41 +1,19 @@
-## Note: This Sass infrastructure is repeated in application-extend1 and application-extend2, but needed in order to address an IE9 rule limit within CSS - http://blogs.msdn.com/b/ieinternals/archive/2011/05/14/10164546.aspx
```

Comment:
@AlasdairSwan I couldn't quite figure out how to abstract the sass shared by the different footer templates.  Happy to revisit this before this is merged, but I think I need some more guidance.